### PR TITLE
Add api to get the keys and the value inside the database

### PR DESCRIPTION
**Pull request by emanlodovice**
Changelog:
* add api to get keys inside the database
* add api to get value given the key

### CODE REVIEW

File: src/LevelDBIntegration.js [hunk 1]
```diff
@@ -53,10 +53,124 @@ function connectToLevelDb(options) {
...
+  ipcMain.on('get-db-keys', (event, params) => {
+    getKeys({ ...params, ipcEvent: event });
+  });
+}
+
+function enableGetValue() {
+  ipcMain.on('get-value-by-key', (event, params) => {
+    getValue({ ...params, ipcEvent: event });
+  });
+}
```
Comment (arnellebalane) [inline src/LevelDBIntegration.js]: @emanlodovice We can merge this into just one IPC listener. We can have only one event e.g `leveldb-command` with the following event params:

```js
{
  command: 'connect',  // or get-keys, get-value, etc.
  params: {
    path: '...'
  }
}
```

We can do a switch-case or whatever in the event handler. That way we don't need to call a lot of functions in the main process.

Comment (emanlodovice) [reply src/LevelDBIntegration.js]: Is this a convention @arnellebalane ?

Comment (arnellebalane) [reply src/LevelDBIntegration.js]: @emanlodovice I think it's more of a design pattern. _Command Pattern_? @jjlumagbas ? 😅

Comment (emanlodovice) [reply src/LevelDBIntegration.js]: updated @arnellebalane

**Comment by emanlodovice** [comment]
from the table UI, I think I will need to add a new endpoint to fetch both keys and values at the same time.

**Comment by arnellebalane** [comment]
@emanlodovice You can do that in a separate PR. Also I just thought, are there places in the app where we would need to fetch just the keys or value, and not both?

**Comment by emanlodovice** [comment]
i guess not 😅 , i was thinking of a firebase like UI where the keys are listed first and the values are fetched when you click one key 😓

**Comment by arnellebalane** [comment]
@emanlodovice Haha, I thought you had something else in mind 😆 We can leave this PR for now until we actually need it. Otherwise, we can close it once we've released the first version of the app without needing these changes 👍

**Comment by emanlodovice** [comment]
hi @arnellebalane , I have updated the PR to be usable already. Please check again

**Pull request merged**